src/Compilers/Core/Portable/DiagnosticAnalyzer/SymbolDeclaredCompilationEvent.cs

```diff
@@ -25,9 +25,11 @@ public SymbolDeclaredCompilationEvent(Compilation compilation, ISymbol symbol, L
         private Lazy<SemanticModel> _lazySemanticModel;
         private SemanticModel _semanticModel;
         private WeakReference<SemanticModel> _weakModel = null;
+        // lockable object only instance is knowledgeable about
```

Comment:
Comment isn't really needed, but XML doc comments would be more appropriate if you're going to keep them. IDE goodness and all. Ditto for everything else.